code/modules/surgery/surgery.dm

```diff
@@ -120,7 +120,7 @@ proc/do_surgery(mob/living/carbon/M, mob/living/user, obj/item/tool)
 	if(user.skills.getRating("surgery") < SKILL_SURGERY_PROFESSIONAL)
 		user.visible_message("<span class='notice'>[user] fumbles around figuring out how to operate [M].</span>",
 		"<span class='notice'>You fumble around figuring out how to operate [M].</span>")
-		var/fumbling_time = SKILL_TASK_FORMIDABLE - ( 6 SECONDS * user.skills.getRating("surgery") ) // 20 secs non-trained, 14 amateur, 8 trained, 2 prof
+		var/fumbling_time = max(1,SKILL_TASK_FORMIDABLE - ( 8 SECONDS * user.skills.getRating("surgery") )) // 20 secs non-trained, 14 amateur, 8 trained, 2 prof
```

Comment:
Max isn't really needed is it?
You could just chaange the check below to `if(fumbling_time && !do_after(user, fumbling_time, TRUE, M, BUSY_ICON_UNSKILLED))`

Reply:
The math I think can go below zero

Reply:
Then perhaps max it to 0 so you can use it as a falsy check. otherwise your doing a 1 decisecond wait, which is wasteful (since you create a progress bar etc, only to remove it the next tick).